### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by shrey27**
Added Features

- Redirect to user's own profile page, by clicking on Profile Page CTA
- Redirect to any random user's profile page, by clicking on the profile pic or username or userid of their post
- Option to edit info
- Add bio to profile
- Check all the posts and sort them between, liked and commented posts

Functions to implement: 
- CRUD operations on posts and edit Profile info options to be added
- Follow or unfollow a user

### CODE REVIEW

**Review by rohit8686 — APPROVED** [review]
- Clean code & Functionality working as expected
- good to merge

File: src/frontend/pages/profile/profile.css [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+.profile__paragraph {
+  font-size: 1.25rem;
+  font-weight: 500;
+  line-height: 1.25rem;
+  text-align: center;
+  margin: 1rem 0;
+}
+.profile__link {
+  font-size: 1.25rem;
+  font-weight: 500;
```
Comment (Logan1x) [inline src/frontend/pages/profile/profile.css:68]: Next time you can make variable for this also

**Review by Logan1x — APPROVED** [review]
Looks good to me.

File: src/App.css [hunk 2]
```diff
@@ -11,7 +11,7 @@
   --background: #e9edf1;
   --white: white;
```
Comment (anuj-kumary) [inline src/App.css:12]: Instead of name use hexadecimal code

**Pull request merged**